packages/fxa-settings/src/pages/ConnectAnotherDevice/index.tsx

```diff
@@ -0,0 +1,249 @@
+                  <FtlMsg id="connect-another-device-signin-confirmed-banner">
+                    <p>Sign-in confirmed</p>
+                  </FtlMsg>
+                </Banner>
+              )}
+            </>
+          )}
+        </>
+      )}
+      <section>
+        <HeartsVerifiedImage />
+        {canSignIn && (
+          <>
+            <FtlMsg id="connect-another-device-signin-to-complete-message">
+              <p className="mb-5 text-sm">
+                Sign in to this Firefox to complete set-up
+              </p>
+            </FtlMsg>
+            <div className="flex">
+              <FtlMsg id="connect-another-device-signin-link">
+                <Link className="cta-primary cta-xl" to={getEscapedSignInUrl()}>
+                  Sign in
+                </Link>
+              </FtlMsg>
+            </div>
+          </>
+        )}
+        {!canSignIn && (
+          <>
+            {device === Devices.FIREFOX_ANDROID && (
+              /* User verified in Fx for Android - they are using an old Fennec
+              or are already signed in. Ignore old browsers, assume already signed in.
+              Encourage installation on another device.
+              */
+              <>
+                {isSignIn ? (
+                  <FtlMsg id="connect-another-device-still-adding-devices-message">
+                    <p className="text-sm">
+                      Still adding devices? Sign in to Firefox on another device
+                      to complete set-up
+                    </p>
+                  </FtlMsg>
+                ) : (
+                  <FtlMsg id="connect-another-device-signin-to-complete-message">
+                    <p className="text-sm">
+                      Sign in to Firefox on another device to complete set-up
+                    </p>
+                  </FtlMsg>
+                )}
+              </>
+            )}
+            {device === Devices.FIREFOX_DESKTOP && isSignedIn && (
+              <>
+                <div className="flex items-center justify-center mt-4 mb-5">
+                  <FtlMsg id="connect-another-device-get-data-on-another-device-message">
+                    <p className="font-normal text-sm px-16">
+                      Want to get your tabs, bookmarks, and passwords on another
+                      device?
+                    </p>
+                  </FtlMsg>
+                </div>
+                <div className="flex">
+                  <FtlMsg id="connect-another-device-cad-link">
+                    <Link
+                      to={`/pair?entrypoint=${entrypoint}`}
+                      className="cta-primary cta-xl"
+                    >
+                      Connect another device
+                    </Link>
+                  </FtlMsg>
+                </div>
+                <div className="mt-5">
+                  <FtlMsg id="connect-another-device-not-now-link">
+                    <Link className="link-blue" to="/settings">
```

Comment:
Should an event be logged here? (`cad.notnow.engage`)

Reply:
It should! I'll add that in